src/animations/animations.css

```diff
@@ -0,0 +1,381 @@
+/* Damage Flash Overlay */
+.damage-flash {
+  position: absolute;
+  top: 0;
+  left: -2px;
+  width: 100%;
+  height: calc(100% - 8px);
+  padding: 0 2px;
+  border-radius: 12px;
+  background-color: rgba(220, 38, 38, 0.5);
+  z-index: 1000;
+  pointer-events: none;
+  animation: damageFlash 0.3s ease-in-out forwards;
+}
+
+@keyframes damageFlash {
+  0% {
+    opacity: 0;
+  }
+  15% {
+    opacity: 1;
+  }
+  75% {
+    opacity: 1;
+  }
+  100% {
+    opacity: 0;
+  }
+}
+
+/* Damage Numeric Label */
+.damage-label {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  font-size: 36px;
+  font-weight: 600;
+  color: white;
+  text-shadow: 1px 1px 0px rgba(0, 0, 0, 0.60);
+  z-index: 1001;
+  pointer-events: none;
+  animation: damageSlideUp 0.3s ease-out forwards;
+}
+
+@keyframes damageSlideUp {
+  0% {
+    transform: translateY(0);
+    opacity: 1;
+  }
+  60% {
+    transform: translateY(-20px);
+    opacity: 1;
+  }
+  100% {
+    transform: translateY(-40px);
+    opacity: 0;
+  }
+}
+
+/* Heal Flash Overlay */
+.heal-flash {
+  position: absolute;
+  top: 0;
+  left: -2px;
+  width: 100%;
+  height: calc(100% - 8px);
+  padding: 0 2px;
+  border-radius: 12px;
+  background-color: rgba(0, 186, 255, 0.5);
+  z-index: 1000;
+  pointer-events: none;
+  animation: healFlash 0.3s ease-in-out forwards;
+}
+
+@keyframes healFlash {
+  0% {
+    opacity: 0;
+  }
+  15% {
+    opacity: 1;
+  }
+  75% {
+    opacity: 1;
+  }
+  100% {
+    opacity: 0;
+  }
+}
+
+/* Heal Numeric Label */
+.heal-label {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  font-size: 36px;
+  font-weight: 600;
+  color: white;
+  text-shadow: 1px 1px 0px rgba(0, 0, 0, 0.60);
+  z-index: 1001;
+  pointer-events: none;
+  animation: healSlideUp 0.3s ease-out forwards;
+}
+
+@keyframes healSlideUp {
+  0% {
+    transform: translateY(0);
+    opacity: 1;
+  }
+  60% {
+    transform: translateY(-20px);
+    opacity: 1;
+  }
+  100% {
+    transform: translateY(-40px);
+    opacity: 0;
+  }
+}
+
+/* Shield Animations */
+
+/* Shield break/defeat animation - explosive shrapnel effect with 4 pieces */
+/* Original shield fades out while shrapnel pieces fly outward */
+
+/* Original shield element - simple fade out */
+.shield-fade-out {
+  animation: shieldFadeOut 0.5s ease-out forwards;
+  pointer-events: none;
+}
+
+@keyframes shieldFadeOut {
+  0% {
+    opacity: 1;
+  }
+  100% {
+    opacity: 0;
+  }
+}
+
+/* Top left piece */
```

Comment:
The shrapnel shield defeat looks good! I think CSS animation code here makes sense too.